www/cordova-plugin-iosrtc.js

```diff
@@ -1,9 +1,9 @@
 /*
- * cordova-plugin-iosrtc v8.0.2
+ * cordova-plugin-iosrtc v8.0.3
```

Comment:
This come from package when generated.
It's not the role of a PR to increment version.

Reply:
Indeed I was very confused why the [CHANGELOG.md](https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/CHANGELOG.md) says 8.0.3 but the `cordova-plugin-iosrtc.js` on the master is 8.0.2.